File: commands/moderation/kick.js
module.exports = {
    name: 'kick',
    permissions: ["KICK_MEMBERS"],
    cooldown: 0,
    execute(client, message, cmd, args, Discord) {

        const target = message.mentions.users.first
        const reason = args.join(' ');

        if (target) {
            const memberTarget = message.guild.members.cache.get(target.id);
            memberTarget.kick();
            message.channel.send({
                embed: { color: `#00f2ff`, description: `${memberTarget} have been kicked! Reasen: *****${reason}*****` }
            })

            const image = message.guild.iconURL({ dynamic: true })
            const userID = message.mentions.users.first()
            const personkicked = client.users.cache.get(userID.id)
            const kickeduser = new Discord.MessageEmbed()
                .setColor('#00f2ff')
                .setTitle('New Warning!')
                .addField('Author', message.author.toString(), true)
                .addField('Guild', message.guild.name, true)
                .addField('Reason', reason)
                .setThumbnail(image)
                .setTimestamp();

            personkicked.send(kickeduser);
        } else {
            message.channel.send({
                embed: { color: `#00f2ff`, description: 'Please enter the name whome to kick' }
            })
        }
    }
}

File: commands/Moderation/kick.py
from setting import *

@commands.has_permissions(kick_members=True)
@client.command()
async def kick(ctx, member : discord.Member, *, reason: str="No reason"):

    # Kicking user
    await member.kick(reason=reason)
    await ctx.send(embed=discord.Embed(title="Kicked a user", description=f"{member.mention} has been kick for `{reason}`", color=0x00f2ff))

# Error(s)
@kick.error
async def kick_error(ctx, error):
    
    # Missing Argument(s)
    if isinstance(error, commands.MissingRequiredArgument):
        embed=discord.Embed(title="Missing Argument(s)", description="Please mention the user to kick!")
        embed.add_field(name="Example", value=f"{prefix}kick {ctx.author.mention}")
        await ctx.send(embed=embed)
    
    # Member not found
    elif isinstance(error, commands.MemberNotFound):
        await ctx.send(embed=discord.Embed(title="Member not found", description="Member you mentioned is not in the server!", color=0x00f2ff))
    
    # Role not high enough
    elif isinstance(error, commands.CommandInvokeError):
        await ctx.send(embed=discord.Embed(title="Role not high enough", description=f"This user is at the top of the role hierarchy then me. Please put me on the top in order to perform this task!", color=0x00f2ff))
    
    # Missing permision
    elif isinstance(error, commands.MissingPermissions):
        await ctx.send(embed=discord.Embed(title="Missing Permision", description=f"Your dont have permission to kick, Noob", color=0x00f2ff))
    
    # Unknown error
    else:
        await unknown_error(ctx, error)